### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by ParameshMuthaiyan** [ParameshMuthaiyan/ContactManager#4]
- [ ] Same name
- [ ] Same number

**Comment by rootanand** [comment]
Same name and number means? Are you going to check for duplicates? How?

**Comment by ParameshMuthaiyan** [comment]
Yes sir, Incase of find the duplicate with same contact name and same mobile number when try to merge the details in given solution.
